### ISSUE TITLE
Expand unit test coverage and expand regression testing

### ISSUE THREAD

**Issue by cbuahin** [USEPA/Stormwater-Management-Model#162]
Set up performance benchmarking and expand unit test coverage for critical portions of the SWMM code. The following areas will be targeted initially for expanding the unit tests:

- SWMM output library (test exists but could be expanded)
- Hotstart file saving and reading
- SWMM API
- SWMM cross-sectional shapes
- Dual drainage implementation

**Comment by dickinsonre** [comment]
By dual drainage you mean the streets and inlets from SWMM 5.2?

**Comment by cbuahin** [comment]
Yes, Robert. It is relatively new and perhaps not as mature as the other portions of the code. Testing it more rigorously will help assure its accuracy going forward.
